src/com/lftechnology/java/training/srijan/basics/iojava/UpperCaseConvertor.java

```diff
@@ -0,0 +1,51 @@
+
+package com.lftechnology.java.training.srijan.basics.iojava;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+
+
+/**
+ * <p>Write a program which reads input one line at a time using a Scanner and
+ * prints it out converted to upper case.</p>
+ * 
+ * @author Srijan
+ */
+public class UpperCaseConvertor {
+	
+	private UpperCaseConvertor(){
+		
+	}
+	
+	private static final Logger LOGGER =
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)